hotline/reports/forms.py

```diff
@@ -119,90 +107,83 @@ def iter_categories(self):
         for category in self.categories:
             yield category, self['category-%d' % category.pk]
 
-    def get_queryset(self):
-        # We do a monster join here so we get all the data we need on all the methods we
-        # call on the report
-        queryset = super().get_queryset().prefetch_related(
-            "image_set",
-            "comment_set__image_set"
-        ).select_related(
-            "reported_category",
-            "reported_species",
-            "actual_species",
-            "actual_species__severity",
-            "reported_species__severity",
-            "actual_species__category",
-            "reported_species__category"
-        )
+    def no_query_found(self):
```

Comment:
This should check if the user isn't active and filter out non-public reports, right? (see patch)